src/main/scala/cromwell/binding/Scope.scala

```diff
@@ -26,45 +50,47 @@ object Scope {
     def getChildrenList(ast: Ast): Seq[Ast] = Option(ast.getAttribute("body")) collect {
```

Comment:
This whole thing was confusing enough that I had to pull it up in intellij and play around w/ the collapsing and such. There's too much going on as one single (logical) line unless some of it is abstracted out into separate functions